Sources/MisticaCommon/Fonts/MisticaFontWeights.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  MisticaFontWeights.swift
+//
+//  Made with ❤️ by Novum
+//
+//  Copyright © Telefonica. All rights reserved.
+//
+
+import SwiftUI
+import UIKit
+
+public protocol MisticaFontWeights {
+    var textPreset4: FontWeightTypable { get }
+    var textPreset5: FontWeightTypable { get }
+    var textPreset6: FontWeightTypable { get }
+    var textPreset7: FontWeightTypable { get }
+    var textPreset8: FontWeightTypable { get }
+    var textPreset9: FontWeightTypable { get }
+    var textPreset10: FontWeightTypable { get }
+}
+
+public enum FontWeightTypable {
```

Comment:
```suggestion
/// Available font weights in Mistica
public enum MisticaFontWeightType {
```

Wdyt?? it's just a set of the allowed weights which will be mapped from the preset ones
https://github.com/Telefonica/mistica-ios/blob/df2b875d290ebf890a28435fb52dbb43755d1d5b/Sources/MisticaCommon/Fonts/FontStyle%2BWeights.swift#L27-L31

and finally mapped to the UIKit weights:
https://github.com/Telefonica/mistica-ios/blob/df2b875d290ebf890a28435fb52dbb43755d1d5b/Sources/MisticaCommon/Fonts/MisticaFontWeights.swift#L28-L39

Reply:
Okey, tbh I wasn't sure with the naming, I like your idea. Fixed!